Components/Navigation/navigation.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Navigation - pigment</title>
+  <link rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.7.2/styles/stackoverflow-dark.min.css"
+    integrity="sha512-9F4w40pQJloG92QgszKGVFODKAkZ70xtcDe2IPcVELXMmkxkojfL6jcP6XSf1NTf4yVc3K0T9h/0gPp8bVIn6w=="
+    crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+    integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" crossorigin="anonymous" />
+  <link rel="stylesheet" href="/CSS/docs.css" />
+  <link rel="stylesheet" href="/CSS/nav.css" />
+  <link rel="stylesheet" href="/CSS/component.css" />
+
+</head>
+
+<body>
+
+  <div class="mobile-nav-button">
+    <div class="brand">
+      <h1><a href="/index.html">Pigment</a></h1>
+
+    </div>
+    <button class="navigation"><i class="fas fa-bars fa-2x"></i></button>
+
+  </div>
+
+  <div class="desktop-main-nav">
+    <div class="brand">
+      <h1><a href="/index.html">Pigment</a></h1>
+    </div>
+
+    <div class="docs">
+      <h3><a href="/docs.html">Docs</a></h3>
+      <span>v.1.0.0</span>
+    </div>
+
+  </div>
+
+  <div class="row ">
+    <div class="side-nav" id="side-nav">
+      <ul>
+        <li><a href="/Components/Avatar/avatar.html">Avatar</a></li>
+        <li><a href="/Components/Alert/alert.html">Alert</a></li>
+        <li><a href="/Components/Badge/badge.html">Badge</a></li>
+        <li><a href="/Components/Button/button.html">Button</a></li>
+        <li><a href="/Components/Card/card.html">Card</a></li>
+        <li><a href="/Components/Image/image.html">Image</a></li>
+        <li><a href="/Components/Input/input.html">Input</a></li>
+        <li><a href="/Components/Lists/list.html">Lists</a></li>
+        <li><a href="/Components/Navigation/navigation.html">Navigation</a></li>
+        <li><a href="/Components/Modal/modal.html">Modal</a></li>
+        <li><a href="/Components/Rating/rating.html">Rating</a></li>
+        <li><a href="/Components/Grids/grid.html">Grid</a></li>
+        <li><a href="/Components/TextUtilities/textUtilities.html">Text Utilities</a></li>
+        <li><a href="/Components/Toast/toast.html">Toast</a></li>
+      </ul>
+    </div>
+
+    <div class="show-content">
+      <section>
+        <h1 class="h1-docs">Navigation</h1>
+        <p class="p-docs"> Navigation is a menu which contains important links to acess the other sections of webpage.
+        </p>
+        <h2>Simple Navbar</2>
+          <div class="nav">
+            <div class="nav-list default">
+              <li><a href="http://">Home</a></li>
+              <li><a href="http://">Products</a></li>
+              <li><a href="http://">About</a></li>
+              <li><a href="http://">Blog</a></li>

```

Comment:
Clicking on the link user is being directed to a blank page.
Instead of using `https://` you can use `javascript:void(0);` this will make it a dead link.

Reply:
Thanks. Made all the changes.